docs/_ext/napdocs.py

```diff
@@ -1,6 +1,7 @@
 """
 Sphinx plugins for nap documentation.
 """
+from __future__ import unicode_literals
```

Comment:
I agree, but it's a small price for pay for better imports and python 3 compatibility. Just think, we can remove them some day once we get upgraded completely!